doc/user_guide/user_guide.md

```diff
@@ -20,16 +20,23 @@ The "mother" of all database objects TDDB creates is the `DatabaseObjectFactory`
 
 Top-level objects are objects in the database which are not scoped by any other objects. A schema for example is a top-level object in Exasol. A table on the other hand lives inside the scope of a schema.
 
-You create a `DatabaseObjectFactory` by choosing an implementation of the concrete database and feeding it with a JDBC connection.
+You choose a database and create a concrete `DatabaseObjectFactory` feeding it with a JDBC connection.
```

Comment:
Also mention that users need to add the dependency for the right JDBC driver(s).

Reply:
added